crates/swc/tests/tsc-references/errorSuperPropertyAccess_es5.1.normal.js

```diff
@@ -144,22 +144,22 @@ var SomeDerived3 = /*#__PURE__*/ function(SomeBase) {
         return _super.apply(this, arguments);
     }
     SomeDerived3.fn = function fn() {
-        _set(_get_prototype_of(SomeDerived3.prototype), "publicStaticMember", 3, this, true);
-        _set(_get_prototype_of(SomeDerived3.prototype), "privateStaticMember", 3, this, true);
+        _set(_get_prototype_of(SomeDerived3), "publicStaticMember", 3, this, true);
```

Comment:
I am not completely sure this change is correct.
But now we get the same result of Babel.

https://babeljs.io/repl/#?browsers=&build=&builtIns=false&corejs=3.21&spec=false&loose=false&code_lz=MYGwhgzhAEDKD2BbApgIUs6BvAUNf0ADgE4CWAbmAC6YkXXIBiArgHbAAUAlNtAL54CdSjSJkRyALLJEAI2TFoAXmgAGANw5B-Qs1khSwInoPAW7brwEFj-w7dPS5C5Ws3ax9URCrV7whlhfKkNzTh4sfg8A72D_cUC44Cd5RRUNLRtdOyMfPyNs0yD8sMtI6yETezyQgqrgYtqUl3T3ARxQSBgEFAARBQpkABMAZmhkAA8aViHupDQMbA8a-wAzVjKPGwhmQgUAOkLDRsNmtOgRzRtt3YOY5BPkmVTXS62CHb3iQ4SaR9KuFcCBV8CsjABzZBUaBgTbXD63b5HBpJM6vIHwz53X4PVHPFoXDHXLFInH_NjhIk2YhQ5jEVjQADkjKJIOgYPZUJhHFYEXeoMRh3qjzRKje8IFXx-Xlx-VFhP57MF93JFkBHgEAiAA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=env%2Ctypescript&prettier=false&targets=&version=7.18.12&externalPlugins=&assumptions=%7B%7D

Reply:
Hmm, maybe previous operation was a bug?